.github/workflows/build_documentation.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Documentation
+
+on:
+  push:
+    branches: [ dev ]
+    paths:
+    - docs/*
+    - metadata_service/*
+    - mkdocs.yml
+
+  workflow_dispatch:
```

Comment:
Why do we need this?

Reply:
That was just a place holder. I haven't added workflow dispatch configurations yet.

This is so that we can trigger workflows from GitHub/cli instead of forcing a trigger event (like a commit).